resources/event-bus/templates/tests/test-e2e-tester.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
 kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: test-core-event-bus-subs
+  name: -test-event-bus-subs
```

Comment:
Is this the convention to follow for charts outside core? (name starting with `-`).
Same question applied for others/

Reply:
Thanks for the catch, no, it's by mistake